src/jira/authenticate.ts

```diff
@@ -22,7 +22,7 @@ export default async (
 	res.locals.installation = installation;
 
 	// TODO: Should the express response logic be inside 'hasValidJwt'?
-	if (hasValidJwt(sharedSecret, jiraHost, req, res)) {
+	if (hasValidJwt(sharedSecret, jiraHost, req, res, TokenType.normal)) {
```

Comment:
Using `jiraHost` url here is wrong and it was passing by accident. The reason is that Jira URL and the app URL have the same path and `atlassian-jwt` ignores the hostname (See https://bitbucket.org/atlassian/atlassian-jwt-js/src/master/lib/jwt.ts#lines-410). However to be correct we need to use the app URL here

[ ] fix base URL to be the app URL